jobs/upload-brew.Jenkinsfile

```diff
@@ -0,0 +1,128 @@
+import org.yaml.snakeyaml.Yaml;
+
+def pipeutils, pipecfg, libcloud, uploading, jenkins_agent_image_tag
+def src_config_url, src_config_ref, brew_principle, brew_profile
+node {
+    checkout scm
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+    libcloud = load("libcloud.groovy")
+
+    brew_principle =  pipecfg['brew']['principle']
+    brew_profile =  pipecfg['brew']['profile']
+}
+
+arches = [ 'aarch64', 'ppc64le', 'x86_64', 's390x']
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'ARCH',
+             description: 'The target architecture',
+             choices: (arches),
+             trim: true),
```

Comment:
when we run this do we only want to upload for a given arch, or do we essentially want all arches for that given build ID to be uploaded to brew?

Assuming we want all arches, can we do it in a single brew call or do we need to do separate brew calls (one for each arch)?

If we need to do separate brew calls I would still consider making this job do them all rather than doing it one by one (mentioned this in https://github.com/coreos/fedora-coreos-pipeline/pull/836/files#r1134578302 too).